vscode-extension/src/util.ts

```diff
@@ -403,15 +398,53 @@ export function validateNewConfigName(name: string, mode: "json" | "yaml") {
   return null;
 }
 
+/**
+ * Some VS Code setups can have multiple workspaces, in which
+ * case we should take the lowest common ancestor path that is shared
+ * across all of them so that the same .env file can be used for multiple
+ * AIConfig files
+ * @returns lowestCommonAncestorPath (string | undefined)
+ *    -> string of path to lowest common ancestor: empty means no shared path
+ *    -> undefined if no workspaces are defined in VS Code session
+ */
+function getLowestCommonAncestorAcrossWorkspaces(): string | undefined {
+  const workspaceFolders = vscode.workspace.workspaceFolders;
+  if (workspaceFolders === undefined || workspaceFolders.length === 0) {
+    return undefined;
+  }
+
+  const workspacePaths = workspaceFolders.map((folder) =>
+    path.normalize(folder.uri.fsPath)
+  );
```

Comment:
nit: probably not needed

Reply:
Future proofing my dude, 100% safety is the name of the game